build.sbt

```diff
@@ -186,12 +186,13 @@ lazy val V = new {
 
   // Scala 2
   def deprecatedScala2Versions = Seq(scala211, "2.12.8", "2.12.9", "2.13.0")
-  def nonDeprecatedScala2Versions = Seq(scala213, scala212, "2.12.10", "2.13.1")
+  def nonDeprecatedScala2Versions =
+    Seq(scala213, scala212, "2.12.10", "2.13.1", "2.13.2")
   def scala2Versions = nonDeprecatedScala2Versions ++ deprecatedScala2Versions
 
   // Scala 3
   def nonDeprecatedScala3Versions = Seq(scala3, "0.24.0")
-  def deprecatedScala3Versions = Seq()
+  def deprecatedScala3Versions = Seq("0.23.0", "0.24.0-RC1")
```

Comment:
Wait, actually, with adding 0.23.0 back in here, don't we also need the `scala-0.23/` directory with ClassFinder in it? If not you get the following when trying to cross publish:
```
[info] Setting Scala version to 0.23.0 on 2 projects.
[info] Excluded 10 projects, run ++ 0.23.0 -v for more details.
[info] Reapplying settings...
[info] set current project to default-86938c (in build file:/Users/ckipp/Documents/scala-workspace/scalameta-org/metals/)
[info] Wrote /Users/ckipp/Documents/scala-workspace/scalameta-org/metals/mtags/target/scala-0.23.0/mtags_0.23.0-0.9.1-SNAPSHOT.pom
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
[info] Compiling 51 Scala sources to /Users/ckipp/Documents/scala-workspace/scalameta-org/metals/mtags/target/scala-0.23.0/classes ...
[error] -- [E008] Member Not Found Error: /Users/ckipp/Documents/scala-workspace/scalameta-org/metals/mtags/src/main/scala-3/scala/meta/internal/pc/CompilerInterfaces.scala:15:34
[error] 15 |import dotty.tools.dotc.reporting.Diagnostic.Error
[error]    |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[error]    |value Diagnostic is not a member of dotty.tools.dotc.reporting - did you mean reporting.diagnostic?
[error] -- [E006] Unbound Identifier Error: /Users/ckipp/Documents/scala-workspace/scalameta-org/metals/mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala:523:8
[error] 523 |        ClassFinder.findClassForOffset(params.offset, filename)(
[error]     |        ^^^^^^^^^^^
[error]     |        Not found: ClassFinder
[error] two errors found
[error] (mtags / Compile / compileIncremental) Compilation failed
```

Reply:
och yeah, forgot about :(

Reply:
I will remove 0.23 back and add only 0.24.0-RC1 to deprecated. With the next upgrade I will try to keep both versions.